server/command.go

```diff
@@ -94,3 +97,11 @@ func getAutoCompleteData() *model.AutocompleteData {
 	aws.AddCommand(listTopics)
 	return aws
 }
+
+func (p *Plugin) getAutoCompleteIconData() string {
+	iconData, err := command.GetIconData(p.API, "assets/icon.svg")
+	if err != nil {
+		errors.Wrap(err, "Failed to get icon data")
```

Comment:
Did you meant to return the error here?

Reply:
Yes, that's right. Is there anything to improve on this code? I just think that error handling in registerCommands() function can be better

Reply:
Maybe a separate function is not even needed. You could use something like: https://github.com/mattermost/mattermost-plugin-github/blob/88aee8396d6dc238fa5507804f3dd4aa751d338a/server/plugin/command.go#L72-L76

Reply:
It looks good. 
Thank you for letting me know the better way. I refactored the code so that a separate function is not needed.